torchtune/utils/collate.py

```diff
@@ -71,3 +71,57 @@ def padded_collate(
             value=padding_idx,
         )
     return input_ids, labels
+
+
+def padded_collate_dpo(
+    batch: List[TokenPair],
+    padding_idx: int = 0,
+    ignore_idx: int = -100,
```

Comment:
nit: could you update this and the normal `padded_collate` to use `torchtune.data._common.CROSS_ENTROPY_IGNORE_IDX`? (this might need to be exposed as public API in `torchtune/data/__init__.py`)